src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.h

```diff
@@ -63,8 +63,13 @@ class OPENMS_DLLAPI FeatureFinderIdentificationAlgorithm :
   /// Main method for actual FeatureFinder
   /// External IDs (@p peptides_ext, @p proteins_ext) may be empty, 
   /// in which case no machine learning or FDR estimation will be performed.
+  /// Optional seeds from e.g. untargeted FeatureFinders can be added with
+  /// @p seeds .
+  /// Results will be written to @p features .
+  /// Caution: peptide IDs will be shrunk to best hit, FFid metavalues added
+  /// and potential seed IDs added.
   void run(
-    std::vector<PeptideIdentification> peptides,
+    std::vector<PeptideIdentification>& peptides,
```

Comment:
do we really need to change peptides? I think this is not a good idea. ALso might not work with new IdentificationDataStructure which is basically read only

Reply:
oops I tried some changes and thought it was like this before. We could save the copy if we would assume that each ID has only one hit and if we remove the hack of adding seeds to the peptide ids.